modules/rubiconBidAdapter.js

```diff
@@ -656,6 +667,17 @@ export function masSizeOrdering(sizes) {
   });
 }
 
+export function determineRubiconVideoSizeId(bid) {
+  // If we have size_id in the bid then use it
+  let rubiconSizeId = parseInt(utils.deepAccess(bid, 'params.video.size_id'));
+  if (!isNaN(rubiconSizeId)) {
+    return rubiconSizeId;
+  }
+  // otherwise 203 for outstream and 201 for instream
+  // When this function is used we know it has to be one of outstream or instream
+  return utils.deepAccess(bidRequest, `mediaTypes.${VIDEO}.context`) === 'outstream' ? 203 : 201;
```

Comment:
Should the argument `bidRequest` be changed to `bid`?

Reply:
yes sir thanks good catch!